qa/automation/ScoringPortal.Tests.Scenarios/3.Sign_in_with_GoogleAcc/1. sign_in.feature

```diff
@@ -0,0 +1,36 @@
+Feature: 3.1 Sign in with Google Account
+
+Background:
+	Given user turn on VPN with necessary config
+	And go to https://qa.ddso-spot.quantori.com/
+
+Scenario: 3.1_01 User has Google account
+	Given User has a saved Google account
+	When User press the button "Sign in with Google"
+	And User select necessary Google account 
+	Then the page with Quantori Scoring is opened
+
+Scenario: 3.1_02 User doesn't have a saved Google account
+	Given User has a google account but it isn't saved in the browser
+	When User press the button "Sign in with Google"
+	And User see the "Sign in to your Google account" field
+	And User fills in all necessary information
+	Then the page with Quantori Scoring is opened
+
+Scenario: 3.1_03 User doesn't have any Google account
+	Given User doesn't have any Google account
+	When User press the button "Sign in with Google"
+	And User see the "Sign in to your Google account" field
+	And fills in the e-mail which is not correspond to any Google account
```

Comment:
It may be a phone number too, and it needs another scenario.

Reply:
We don't need to test google sign-in itself